app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,46 @@
+class PostsController < ApplicationController
+  before_action :authenticate_user!
+  def index
+
+  end
+
+  def new
+    @post = current_user.posts.build
+  end
+
+  def create
+    @post = current_user.posts.build(post_params)
+    if @post.save
+      flash[:notice] = 'Post successfully created'
+      redirect_to post_path(@post)
+    else
+      render 'new'
+    end
+  end
+
+  def show
+    @post = Post.find(params[:id])
+  end
+
+  def destroy
+    @post = Post.find(params[:id])
```

Comment:
current_user.posts.find ...

Now you can delete any post :)

Reply:
fixed